### PR TITLE
Feat/pages

### PR DESCRIPTION
**Pull request by yoslie**
### Overview
Slicing layout feature pages

### Changes
- add pages parent index
- add pages children index
- create tab menu componente
- update header page title
- refactor stlyle layout

### Preview
image

### Related
[S5A4 - Page Status (Slicing) - Menu Halaman](https://app.clickup.com/t/860rumb4a)

### Evidence
title: slicing layout structure index menu halaman
project: J-Site
participants: @yoslie @Ibwedagama @marsellavaleria19

### CODE REVIEW

File: components/TabMenu.vue [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        </NuxtLink>
+      </Tab>
+    </TabList>
+  </TabGroup>
+</template>
+
+<script setup lang="ts">
+  import { TabGroup, TabList, Tab } from '@headlessui/vue'
+
+  const route = useRoute()
```
Comment (Ibwedagama) [inline components/TabMenu.vue:30]: hello @yoslie is this code used?

File: components/TabMenu.vue [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  interface ITab {
+    label: string
+    link: string
+  }
+
+  defineProps({
+    tabs: {
+      type: Array<ITab>,
```
Comment (Ibwedagama) [inline components/TabMenu.vue:39]: hello @yoslie I think this should be

```
type: Array as PropType<ITab[]>
```

File: components/TabMenu.vue (hunk 1 above)
Comment (yoslie) [reply components/TabMenu.vue:30]: it is, to detect actived tabs `route.path === tab.link`

File: components/TabMenu.vue (hunk 2 above)
Comment (yoslie) [reply components/TabMenu.vue]: 😮‍💨 updated [here](https://github.com/jabardigitalservice/j-site-builder/pull/16/commits/95e77a47c932e37b54ecefa4930a7089d28e859d), thanks @Ibwedagama

**Review by marsellavaleria19 — APPROVED** [review]

**Review by Ibwedagama — APPROVED** [review]

**Pull request merged**